### PR TITLE
Update ua.json

### PR DESCRIPTION
**Pull request by Apenflex**
Hi. I made some changes in translation.

### CODE REVIEW

**Comment by knowbody** [comment]
Hey, thanks a lot for your contribution! It seems like you removed quite a few translations. See the "files changed" tab. so we cannot accept this PR until that is fixed.

File: ua.json [hunk 1]
```diff
@@ -1554,195 +1565,16 @@
...
-  "saved_destination_charge_at_location": "Зарядка в цьому місці",
-  "saved_destination_charge_at_location_description": "Ця інформація є приватною та зберігається лише у вашому обліковому записі.",
-  "open_app_settings": "Відкрити налаштування програми",
-  "ignore_once": "Ігнорувати до наступного старту",
-  "grant_automotive_permissions": "Перейдіть до налаштувань додатка і переконайтеся, що дозволи „Місцезнаходження“ та „Інформація про автомобіль“ надані.",
-  "share_apple_link": "Відкрити в Apple Maps",
-  "delete_all_activities": "Видалити збережені активності",
-  "delete_all_activities_description": "Усі збережені активності, пов'язані з цим автомобілем, будуть видалені.",
-  "delete_all_activities_dialog": "Ви впевнені, що хочете видалити всі активності, пов'язані з цим автомобілем?",
-  "delete_activity": "Видалити активності",
```
Comment (knowbody) [inline ua.json:1576]: we need all of these back

**Review by knowbody — CHANGES_REQUESTED** [review]

**Comment by Apenflex** [comment]
Ooo that's a problem I should check why all this lines have been gone. I
will fix and create new PR.

пт, 29 груд. 2023, 12:04 користувач Mateusz Zatorski <
***@***.***> пише:

**Comment by knowbody** [comment]
ok thank you, closing this one then.

**Pull request closed without merging**